src/serac/numerics/functional/tuple_arithmetic.hpp

```diff
@@ -202,6 +202,30 @@ SERAC_HOST_DEVICE auto promote_to_dual_when(const T& x)
   }
 }
 
+/**
+ * @tparam dualify specify whether or not the input should be made into its dual type
+ * @tparam T the type of the values passed in
+ * @tparam n how many values were passed in
+ *
+ * @brief a function that optionally (decided at compile time) converts a list of values to their dual types
```

Comment:
I kind of like the brief at the top. Is this the normal way we do it in the code?

Reply:
I've moved all the `@briefs` I could find to the top (except for the `@file` comments, where we seem to be consistently putting the `@brief` lower down)